q2_sample_classifier/utilities.py

```diff
@@ -844,11 +854,9 @@ def _warn_feature_selection():
 
 
 def _warn_zero_test_split():
-    warning = (
+    return \
```

Comment:
you could drop the `\` and also the parenthesis for a cleaner look 😎

Reply:
done